terraform/github/teams.tf

```
@@ -250,6 +264,13 @@ locals {
       ]
     }
 
+    asdf-gitlab-lab = {
+      description = "The people with push access to the asdf-lab repository"
```

```suggestion
    asdf-glab = {
      description = "The people with push access to the asdf-glab repository"
```

If you intend this: https://gitlab.com/gitlab-org/cli

---

No that's another tool (which I also have a repo for). One is called Lab and the other is GLab.

---

OK!